apps/token-manager/app/src/components/AutoComplete/AutoCompleteSelected.js

```diff
@@ -0,0 +1,108 @@
+import React, { useRef, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { ButtonBase, theme } from '@aragon/ui'
+import AutoComplete from './AutoComplete'
+
+const identity = x => x
+const noop = () => null
+
+const AutoCompleteSelected = React.memo(
+  React.forwardRef(function AutoCompleteSelected(
+    {
+      itemButtonStyles,
+      items,
+      onChange,
+      onSelect,
+      onSelectedClick = noop,
+      renderItem,
+      required,
+      renderSelected = identity,
+      selected,
```

Comment:
What is the role of `selected` and how does it differ from `value`?

Reply:
I saw below: `value` is the search term(s) and selected is the selected identity (or any other type) object

Reply:
> I saw below: value is the search term(s) and selected is the selected identity (or any other type) object

Yes. This autocomplete components allow for rendering on complexer objects, but in the end it works on an input element so had to keep value and used selected (which for simple elements will probably just be some text, or the dev can use `AutoComplete` without `AutoCompleteSelected`.